routes/index.js

```diff
@@ -1,12 +1,20 @@
-const express = require('express')
+const express = require('express');
 const router = express.Router();
-const usersRouter = require('./users')
+const crud = require('../services/crud');
+const usersRouter = require('./users');
+const Room = require('../models/room');
+const Booking = require('../models/booking');
+const Image = require('../models/image');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
   res.json({ title: 'Welcome' });
 });
 
-usersRouter(router)
+usersRouter(router);
+
+router.use('/rooms', crud(Room));
```

Comment:
As usersRouter, I think I would have created a file roomsRouter, even thought there is only one line ( router.use('/rooms', crud(Room)); ) so that if there is some special route, it would fit just well in it. ( Same for bookings and images)